lecture_3/03_project/calculator/internal/tasks/signalHandler.go

```diff
@@ -0,0 +1,27 @@
+package tasks
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+// SignalHandler is handling incoming OS signals.
+type SignalHandler struct{}
```

Comment:
if you copy/pasted the signal handler then you do not need the dependency that is mentioned above